packages/macro/package.json

```diff
@@ -63,7 +63,7 @@
     "@lingui/message-utils": "4.0.0"
   },
   "peerDependencies": {
-    "@lingui/react": "4.0.0",
+    "@lingui/react": "^4.0.0",
```

Comment:
All Lingui packages use an exact dependency version of other Lingui packages. Do we really need this change?

Reply:
okay, I'll revert